operators/src/processing/point_in_polygon/tester.rs

```diff
@@ -130,6 +172,14 @@ impl<'a> PointInPolygonTester<'a> {
         let polygon_offsets = self.polygon_offsets;
         let ring_offsets = self.ring_offsets;
 
+        let polygon_bounding_box = &self.polygon_bounds[feature_index];
+        if !polygon_bounding_box
+            .iter()
+            .any(|b| b.contains_coordinate(&coordinate))
```

Comment:
I think it would make sense to precompute the multi polygon bounds as well so you need only one check here

Reply:
i added that:  [81ff742](https://github.com/geo-engine/geoengine/pull/792/commits/81ff742edf53e417c4c48d9c70684e9ba3889e51)